### PR TITLE
Add spider for Coffee#1

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

**Comment by Cj-Malone** [comment]
Hey! Welcome :)

https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/6849154694/coffee_1_gb/output.geojson

There is a bit of a mistake with internet access tag, you should checkout apply_yes_or_no

File: locations/spiders/coffee_1_gb.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        else:
+            address = location["branch_address"].split(", ")
+
+        if "branch-fallback" not in location["branch_image"]:
+            item["image"] = location["branch_image"]
+
+        item["ref"] = re.search(r"/locations/([^/]+)/$", location["url"]).group(1)
+        item["name"] = location["post_title"]
+        item["city"] = location["branch_city"]
+        item["addr_full"] = "\n".join(address)
```
Comment (Cj-Malone) [inline locations/spiders/coffee_1_gb.py:37]: We typically split addr_full with commas.

```suggestion
        item["addr_full"] = ", ".join(address)
```

**Review by Cj-Malone — APPROVED** [review]
Otherwise LGTM.

I don't know why it's only getting 1 on the CI, but it's working locally:

```python
{'atp/brand/Coffee#1': 115,
 'atp/brand_wikidata/Q22032058': 115,
 'atp/category/amenity/cafe': 115,
 'atp/field/country/from_spider_name': 115,
 'atp/field/email/missing': 115,
 'atp/field/image/missing': 6,
 'atp/field/phone/invalid': 1,
 'atp/field/phone/missing': 2,
 'atp/field/state/missing': 115,
 'atp/field/street_address/missing': 115,
 'atp/field/twitter/missing': 115,
 'atp/nsi/perfect_match': 115,
 'downloader/request_bytes': 42982,
 'downloader/request_count': 117,
 'downloader/request_method_count/GET': 117,
 'downloader/response_bytes': 1031847,
 'downloader/response_count': 117,
 'downloader/response_status_count/200': 117,
 'elapsed_time_seconds': 30.510689,
 'feedexport/success_count/FileFeedStorage': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2023, 11, 13, 22, 7, 13, 39940),
 'httpcache/firsthand': 26,
 'httpcache/hit': 91,
 'httpcache/miss': 26,
 'httpcache/store': 26,
 'httpcompression/response_bytes': 5948259,
 'httpcompression/response_count': 117,
 'item_scraped_count': 115,
 'log_count/DEBUG': 244,
 'log_count/INFO': 10,
 'log_count/WARNING': 1,
 'memusage/max': 149733376,
 'memusage/startup': 149733376,
 'request_depth_max': 1,
 'response_received_count': 117,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 116,
 'scheduler/dequeued/memory': 116,
 'scheduler/enqueued': 116,
 'scheduler/enqueued/memory': 116,
 'start_time': datetime.datetime(2023, 11, 13, 22, 6, 42, 529251)}
```

**Comment by EdwardBetts** [comment]
> Otherwise LGTM.
> 
> I don't know why it's only getting 1 on the CI, but it's working locally:
> 
> ```python
> {'atp/brand/Coffee#1': 115,
>  'atp/brand_wikidata/Q22032058': 115,
>  'atp/category/amenity/cafe': 115,
>  'atp/field/country/from_spider_name': 115,
>  'atp/field/email/missing': 115,
>  'atp/field/image/missing': 6,
>  'atp/field/phone/invalid': 1,
>  'atp/field/phone/missing': 2,
>  'atp/field/state/missing': 115,
>  'atp/field/street_address/missing': 115,
>  'atp/field/twitter/missing': 115,
>  'atp/nsi/perfect_match': 115,
>  'downloader/request_bytes': 42982,
>  'downloader/request_count': 117,
>  'downloader/request_method_count/GET': 117,
>  'downloader/response_bytes': 1031847,
>  'downloader/response_count': 117,
>  'downloader/response_status_count/200': 117,
>  'elapsed_time_seconds': 30.510689,
>  'feedexport/success_count/FileFeedStorage': 1,
>  'finish_reason': 'finished',
>  'finish_time': datetime.datetime(2023, 11, 13, 22, 7, 13, 39940),
>  'httpcache/firsthand': 26,
>  'httpcache/hit': 91,
>  'httpcache/miss': 26,
>  'httpcache/store': 26,
>  'httpcompression/response_bytes': 5948259,
>  'httpcompression/response_count': 117,
>  'item_scraped_count': 115,
>  'log_count/DEBUG': 244,
>  'log_count/INFO': 10,
>  'log_count/WARNING': 1,
>  'memusage/max': 149733376,
>  'memusage/startup': 149733376,
>  'request_depth_max': 1,
>  'response_received_count': 117,
>  'robotstxt/request_count': 1,
>  'robotstxt/response_count': 1,
>  'robotstxt/response_status_count/200': 1,
>  'scheduler/dequeued': 116,
>  'scheduler/dequeued/memory': 116,
>  'scheduler/enqueued': 116,
>  'scheduler/enqueued/memory': 116,
>  'start_time': datetime.datetime(2023, 11, 13, 22, 6, 42, 529251)}
> ```

I hadn't included any refs, that why there was only one on the CI. I've fixed it now.

**Comment by Cj-Malone** [comment]
Ah, yes it's working on latest version https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/6856792231/coffee_1_gb/output.geojson (CI times out after ~60 seconds)

**Pull request merged**